### PR TITLE
fix(components/indicators): sky-wait restore focus when wait closes

### PR DESCRIPTION
**Pull request by johnhwhite**
[AB#2169841](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/2169841)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/377/storybook/)

Component Storybooks:

- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/377/storybooks/indicators-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/377/code-examples/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/377/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/377/playground/)

File: libs/components/indicators/src/lib/modules/wait/wait.component.ts [hunk 1]
```diff
@@ -36,6 +36,25 @@ export class SkyWaitComponent implements OnInit {
       this.adapterService.removeWaitBounds(this.elRef);
     }
 
+    const blockFocus = this._isFullPage || !this.isNonBlocking;
+    if (blockFocus) {
+      if (value) {
+        // Before wait, capture the active element.
+        if (
+          (this.elRef.nativeElement.parentElement as HTMLElement).matches(
```
Comment (Blackbaud-TrevorBurch) [inline libs/components/indicators/src/lib/modules/wait/wait.component.ts]: Any DOM calls should probably be in the adapter

**Review by Blackbaud-TrevorBurch — CHANGES_REQUESTED** [review]
@johnhwhite one small thing

File: libs/components/indicators/src/lib/modules/wait/wait.component.ts (hunk 1 above)
Comment (johnhwhite) [reply libs/components/indicators/src/lib/modules/wait/wait.component.ts]: Moved the changes to the adapter.

**Review by Blackbaud-TrevorBurch — APPROVED** [review]

**Pull request merged**